akka-http-core/src/main/scala/akka/http/Http.scala

```diff
@@ -99,27 +99,34 @@ class HttpExt(config: Config)(implicit system: ActorSystem) extends akka.actor.E
     bindAndHandle(Flow[HttpRequest].mapAsync(handler), interface, port, backlog, options, settings, log)
 
   /**
-   * Transforms a given HTTP-level server [[Flow]] into a lower-level TCP transport flow.
+   * The type of the server-side HTTP layer as a stand-alone BidiStage
+   * that can be put atop the TCP layer to form an HTTP server.
+   *
+   * {{{
+   *                +------+
+   * HttpResponse ~>|      |~> ByteString
```

Comment:
What should be the convention? Lower layer to the left or right? I used lower layer to the left before, but anything is OK, but it would be nice to establish a convention now.